README.md

```diff
@@ -1,296 +1,93 @@
-
-1. Navigate to the explorer tree.
-2. Open the **DATA SETS** bar.
-3. Select a data set you want to rename.
-4. Right-click the data set and select the **Rename Data Set** option.
-5. Change the name of the data set.
-
-![Rename Data Set](/docs/images/ZE-rename.gif?raw=true "Rename Data Set")
-<br /><br />
-
-### Copy data sets
-
-1. Navigate to the explorer tree.
-2. Open the **DATA SETS** bar.
-3. Select a member you want to copy.
-4. Right-click the member and select the **Copy Data Set** option.
-5. Right-click the data set where the member belongs and select the **Paste Data Set** option.
-6. Enter the name of the copied member.
-
-![Copy Data Set](/docs/images/ZE-copy.gif?raw=true "Copy Data Set")
-<br /><br />
-
-### Download, edit, and upload existing PDS members
-
-1. Navigate to the explorer tree.
-2. Open the **DATA SETS** bar.
-3. Open a profile.
-4. Click the PDS member (or PS) that you want to download.
-
-   **Note:** To view the members of a PDS, click the PDS to expand the tree.
-
-   The PDS member is displayed in the text editor window of VSC.
-
-5. Edit the document.
-6. Navigate back to the PDS member (or PS) in the explorer tree, and click the **Save** button.
-
-Your PDS member (or PS) is uploaded.
-
-**Note:** If someone else has made changes to the PDS member (or PS) while you were editing it, you can merge your conflicts before uploading to the mainframe.
+- Install [Yarn](https://yarnpkg.com/getting-started/install).
 
-![Edit](/docs/images/ZE-download-edit.gif?raw=true "Edit")
-<br /><br />
+## Directory Structure
 
-### Use the save option to prevent merge conflicts
+Zowe Explorer repository includes several folders with files that let you build and configure various aspects of the extension. The bulk of the Zowe Explorer source code is in the `packages` directory. The `packages` folder has the following structure:
 
-1. Navigate to the explorer tree.
-2. Open the **DATA SETS** bar.
-3. Open a member of a data set you want to edit.
-4. Edit a data set.
-5. Press Ctrl+S or Command+S (OSx) to save you changes.
-6. (Optional) Resolve merge conflicts if necessary.
+- `├──`[`packages`](./packages) — includes source code of Zowe Explorer, various Zowe Explorer-related extensions, and the extensibility API<br>
+    - `├──`[`eslint-plugin-zowe-explorer`](./packages/eslint-plugin-zowe-explorer) — includes necessary files to configure ESLint plug-in for Zowe Explorer
+    - `├──`[`zowe-explorer-api`](./packages/zowe-explorer-api) — includes files to set up and use Extensibility API for Explorer. The API has two modules: Profiles API and Tree API.
+    - `├──`[`zowe-explorer-ftp-extension`](./packages/zowe-explorer-ftp-extension) — includes files to set up and use the FTP extension for Zowe Explorer. 
+    - `├──`[`zowe-explorer`](./packages/zowe-explorer) — core Zowe Explorer source files, ReadMe, Changelog, and more.
 
-![Save](/docs/images/ZE-safe-save.gif?raw=true "Save")
-<br /><br />
+## Build Locally
```

Comment:
Should this "Build Locally" section be moved under the Zowe Explorer package's readme? The instructions are specific to building that vsix file.

Reply:
I think the rationale behind having this section in this readme is to provide developers with an excerpt from the Dev's readme and include the link to the Dev's readme at the same time.
Actually, originally this section was part of the core ZE readme (package's readme now) but we decided to exclude it from there in order not to overload users with more options to install the extension.

Reply:
Personally, I think it makes more sense to put this under the Zowe Explorer package's readme. However, if the squad prefers to keep it in the main repo readme, then perhaps we could add a line specifying that these instructions are just for building the Zowe Explorer VS Code extension `.vsix` file.